frontends/p4/tableKeyNames.cpp

```diff
@@ -31,6 +33,9 @@ void KeyNameGenerator::postorder(const IR::PathExpression *expression) {
 
 namespace {
 
+static const cstring isValid = "isValid"_cs;
+static const cstring isValidD = "$valid$"_cs;
```

Comment:
Maybe add a comment what this is or use a more explaining name? I don't see what the `D` in the name stands for.